framework/decode/vulkan_object_info.h

```diff
@@ -647,33 +647,10 @@ struct VulkanRenderPassInfo : public VulkanObjectInfo<VkRenderPass>
     std::vector<VkImageLayout>           attachment_description_final_layouts;
     std::vector<VkAttachmentDescription> attachment_descs;
 
-    struct SubpassReferences
-    {
-        VkSubpassDescriptionFlags          flags;
-        VkPipelineBindPoint                pipeline_bind_point;
-        std::vector<VkAttachmentReference> input_att_refs;
-        std::vector<VkAttachmentReference> color_att_refs;
-        std::vector<VkAttachmentReference> resolve_att_refs;
-        std::vector<uint32_t>              preserve_att_refs;
-
-        bool                  has_depth;
-        VkAttachmentReference depth_att_ref;
-    };
-
-    // The attachment references per subpass
-    std::vector<SubpassReferences> subpass_refs;
+    std::vector<uint8_t> create_info;
```

Comment:
I guess there's not much cost to having two vectors here if one is always empty, but could this just be one vector?  Is whether one is empty ever used in conditionals?

Reply:
> Is whether one is empty ever used in conditionals?
Yes. The one that is not empty decides which function to use, `vkCreateRenderPass` or `vkCreateRenderPass2`